batch.md

```diff
@@ -48,52 +62,76 @@ DataSource<EventType> dataSet = env.createInput(inputFormat, TypeInformation.of(
+Each stream in Pravega is contained by a scope.  A scope acts as a namespace for one or more streams. The [BatchClient](https://github.com/pravega/pravega/blob/master/client/src/main/java/io/pravega/client/batch/BatchClient.java) is able to read from numerous streams in parallel, even across scopes.  The builder API accepts both **qualified** and **unqualified** stream names.  
+
+  - In aualified stream names, the scope is explicitly specified, e.g. `my-scope/my-stream`. 
+  - In unqualified stream names are assumed to refer to the default scope as set in the `PravegaConfig`.
 
 A stream may be specified in one of three ways:
 1. As a string containing a qualified name, in the form `scope/stream`.
 2. As a string containing an unqualified name, in the form `stream`.  Such streams are resolved to the default scope.
 3. As an instance of `io.pravega.client.stream.Stream`, e.g. `Stream.of("my-scope", "my-stream")`.
 
-### Stream Cuts
-A `StreamCut` represents a specific position in a Pravega stream, which may be obtained from various API interactions with the Pravega client.   The [BatchClient](https://github.com/pravega/pravega/blob/master/client/src/main/java/io/pravega/client/batch/BatchClient.java) accepts a `StreamCut` as the start and/or end position of a given stream. For further reading on Stream Cuts, you could refer to this  [documentation](https://github.com/pravega/pravega/blob/master/documentation/src/docs/streamcuts.md) and/or a [sample code](https://github.com/pravega/pravega-samples/tree/v0.3.2/pravega-client-examples/src/main/java/io/pravega/example/streamcuts).
+### StreamCuts
 
-If unspecified, the default start position is the earliest available data.  The default end position is all available data as of when job execution begins.
+A `StreamCut` represents a specific position in a Pravega stream, which may be obtained from various API interactions with the Pravega client. The [BatchClient](https://github.com/pravega/pravega/blob/master/client/src/main/java/io/pravega/client/batch/BatchClient.java) accepts a `StreamCut` as the start and/or end position of a given stream.  For further reading on StreamCuts, please refer to documentation on [StreamCut](https://github.com/pravega/pravega/blob/master/documentation/src/docs/streamcuts.md) and [sample code](https://github.com/pravega/pravega-samples/tree/v0.3.2/pravega-client-examples/src/main/java/io/pravega/example/streamcuts).
 
+If unspecified, the default start position is the earliest available data. The default end position is all available data as of when job execution begins.
+(can u rephrase??)

```

Comment:
If stream cuts are not provided then the default start position requested is assumed to be the earliest available data in the stream and the default end position is assumed to be all available data in that stream as of when the job execution begins.

Reply:
Accepted suggestion